plugins/plugin-java-debugger/che-plugin-java-debugger-server/src/test/java/org/eclipse/che/plugin/jdb/server/BreakpointConditionTest.java

```diff
@@ -66,19 +60,32 @@ public void tearDown() throws Exception {
   }
 
   @Test
+  public void shouldStopByHitCount() throws Exception {
+    debugger.addBreakpoint(
+        new BreakpointImpl(
+            new LocationImpl("/test/src/org/eclipse/BreakpointsByConditionTest.java", 19, "/test"),
+            true,
+            new BreakpointConfigurationImpl(3)));
+
+    DebuggerEvent debuggerEvent = events.take();
+    assertTrue(debuggerEvent instanceof BreakpointActivatedEvent);
+
+    debugger.resume(new ResumeActionImpl());
+
+    debuggerEvent = events.take();
+    assertTrue(debuggerEvent instanceof SuspendEvent);
+
+    assertEquals("2", debugger.evaluate("i"));
+    assertEquals("2", debugger.evaluate("k"));
+  }
+
+  @Test(priority = 1)
   public void shouldStopByCondition() throws Exception {
     Breakpoint breakpoint =
         new BreakpointImpl(
-            new LocationImpl(
-                "/test/src/org/eclipse/BreakpointsByConditionTest.java",
-                19,
-                false,
-                -1,
-                "/test",
-                null,
-                -1),
+            new LocationImpl("/test/src/org/eclipse/BreakpointsByConditionTest.java", 24, "/test"),
```

Comment:
Do we have a test case for invalid condition?

Reply:
No.

Reply:
Maybe it will be good to add?